test/unittest/tests/block/test_extent_list_search.cpp

```diff
@@ -0,0 +1,298 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * [extent_list2]: block_ext.c
+ * Test extent list functions part 3. (More to come.)
+ *
+ * Test extent list search functions: __block_off_srch_pair, and __block_off_match.
+ */
+
+#include <algorithm>
+#include <catch2/catch.hpp>
+#include <memory>
+#include <sstream>
+#include <string>
+
+#include "wt_internal.h"
+#include "../wrappers/mock_session.h"
+#include "test_util.h"
+#include "../utils.h"
+#include "../utils_extlist.h"
+
+using namespace utils;
+
+/*!
+ * A test (_off) and expected values (_before and _after) for __block_srch_pair.
+ */
+struct search_before_after {
+    wt_off_t _off;
```

Comment:
Struct member variables don't have an _. If this was a class this would be correct. However I think we should probably revisit our decision around public ones being `_var_name`.

Reply:
Done